en/admin/migration.inc

```diff
@@ -166,6 +168,26 @@ The following is [options] used in **cubrid unloaddb**.
    
     This option makes it to unload the latest image of objects in data volumes when unloading instances (the MVCC version is ignored and it does not refer to the log volumes). It may include uncommitted data and deleted data that has not been vacuumed.
 
+.. option:: -t, --thread-count=COUNT
+
+    It is performed in parallel using as many threads as the number of COUNT, and COUNT must be in the range of 0 to 127.
+    If the setting is omitted, it is the same as specifying 1, and if it is specified as 0, it does not operate in thread mode.
+    Even if the thread mode is specified, if the table contains an Object type or a Set, MultiSet, or Sequence (List) type that contains an Object type, it will not operate in thread mode for that table.
+    
+    ::
+
+        cubrid unloaddb -t 4 demodb
+
+.. option:: --enhanced-estimates
+
+    -v option must be specified together.
+    When determining the number of total target records, actual values ​​are obtained instead of statistical information. Note that when this option is specified, the execution time to determine the number of records may be longer.
```

Comment:
```suggestion
    When determining the number of records in the target table, obtain the actual number of records instead of statistical information. If this option is specified, the unload execution time may be longer because the execution time to calculate the number of records is added, so it must be used with caution.
```